### ISSUE TITLE
Activitybar border missing in Light HC theme

### ISSUE THREAD

**Issue by hbons** [microsoft/vscode#205100]
The 2px wide blue border for the active View is missing in the Light HC theme.

<img width="66" alt="image" src="https://github.com/microsoft/vscode/assets/103326/435b0c11-25fe-433f-9cf7-ca36ccb721a7">

**Comment by sandy081** [comment]
Not showing the active indicator in HC themes is intentional based on the comments in the code. Showing this in HC Black theme is not expected.

https://github.com/microsoft/vscode/blob/5bc6776982eb94191421fd211edf316a6bf6cb4a/src/vs/workbench/browser/parts/activitybar/media/activityaction.css#L121

@hbons I will leave it to up to you to either remove it for HC themes or add it.

**Comment by hbons** [comment]
@daviddossett when you're back: do you know why this was removed?

**Comment by daviddossett** [comment]
Nope, not sure why this was removed. My best guess is that the hover/active styling used on the bottom activity bar items was intended to take over in place of the border-left in HC themes. It's a known issue that the CSS isn't working for that case.

![Image](https://github.com/microsoft/vscode/assets/25163139/094f854a-e955-4958-8ba3-4af920fa6e7f)
